dali/test/python/test_pipeline_debug.py

```diff
@@ -535,13 +548,13 @@ def test_variable_batch_size_from_external_source():
 
 @pipeline_def(batch_size=8, num_threads=3, device_id=0, seed=47, debug=True)
 def incorrect_variable_batch_size_from_es_pipeline():
-    rng = fn.random.coin_flip(probability=0.5)
     src_data = np.zeros((1, 6, 64, 64, 3), dtype=np.uint8)
     images = fn.external_source(src_data)
     return images,
 
 
-@raises(RuntimeError, glob='Batch size must be uniform across an iteration. External Source operator returned batch size*')
+@raises(RuntimeError, glob=('Batch size must be uniform across an iteration.'
+                            ' External Source operator returned batch size*'))
```

Comment:
```suggestion
@raises(RuntimeError, glob='Batch size must be uniform across an iteration.'
                           ' External Source operator returned batch size*')
```

Reply:
I would agree, but I would need to change almost whole file for it to work, or place the some escaped singe `'` in the one that uses `"`, so I would say it's not worth it.

Reply:
I was just suggesting it only because this line is already being changed.